Makefile

```diff
@@ -81,11 +81,18 @@ $(GENERATE_UNBOUND_METHODS_EXAMPLE_SRCS): $(GENERATE_UNBOUND_METHODS_EXAMPLE_SPE
 	@rm -f $(EXAMPLE_CLIENT_DIR)/generateunboundmethods/README.md \
 		$(EXAMPLE_CLIENT_DIR)/generateunboundmethods/git_push.sh
 
+TMP_INSTALL_DIR := $(shell mktemp -d)
 install:
+
+install:
+	@mkdir -p ${TMP_INSTALL_DIR}
+	cd ${TMP_INSTALL_DIR} && go get \
+		google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.0.0 \
+		google.golang.org/protobuf/cmd/protoc-gen-go@v1.26.0 \
+		github.com/bufbuild/buf/cmd/buf@v0.41.0
+	@rmdir ${TMP_INSTALL_DIR}
```

Comment:
Does this command still run if the previous one failed?

Reply:
If the `mkdir` fails then the make process will exit with an error code. So the `rmdir` won't try to run. Actually if any of the 3 commands fail it will terminate the make process with an error code.

Reply:
That's what I mean, can we "trap" the rmdir so that it doesn't stick around if the installation fails?

Reply:
Ah but the temp dir will be something like this `/tmp/tmp.4PHFyK4afw` so if it does not get cleaned up it is not too bad. Most systems will clean that up (eventually). Not sure if it is worth adding the complexity. Do you have an elegant suggestion?

Reply:
Nah it's not the end of the world, you're right.